.github/workflows/nextstrain-docker-base.yml

```diff
@@ -0,0 +1,39 @@
+# Trigger rebuild in https://github.com/nextstrain/docker-base
+name: nextstrain-docker-base
+
+on:
+  release:
+    types:
+      - released
+
+  repository_dispatch:
+    types: nextstrain-docker-base
+
+  workflow_dispatch:
+
+  workflow_call:
```

Comment:
No need for these. The only time this should run is on a new Nextclade release (`release` trigger). The image can be rebuilt manually using `workflow_dispatch` on [docker-base/CI](https://github.com/nextstrain/docker-base/actions/workflows/ci.yml).